### PR TITLE
feat(quantic): added ariaLabel for quanticNumberButton

### PR DESCRIPTION
**Pull request by MSLI-Coveo**
[SVCC-2300](https://coveord.atlassian.net/browse/SVCC-2300)
Added ariaLabel for quantic
Screen Shot 2022-08-18 at 4 21 37 PM
NumberButton
Screen Shot 2022-08-18 at 4 23 22 PM

### CODE REVIEW

File: packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 1]
```diff
@@ -147,6 +147,34 @@
...
+        <protected>false</protected>
+        <shortDescription>Go to page ...</shortDescription>
+    </labels>
+    <labels>
+        <fullName>quantic_ShowNResultsPerPage</fullName>
+        <value>Show {{0}} per page</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Show N number of results per page</shortDescription>
+    </labels>
```
Comment (erocheleau) [inline packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml:177]: @acarpentier01 String revision on these labels please :)

File: packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 2]
```diff
@@ -147,6 +147,34 @@
...
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Go to page ...</shortDescription>
+    </labels>
+    <labels>
+        <fullName>quantic_ShowNResultsPerPage</fullName>
+        <value>Show {{0}} per page</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Show N number of results per page</shortDescription>
```
Comment (erocheleau) [inline packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml:176]: Your value and shortDescription don't exactly match right?

1 is `show 5 per page` while the description says : `show 5 number of results per page` ?

If possible we should be saying `show 5 results per page`

**Review by erocheleau — APPROVED** [review]
Small string change and I tagged Anny for string revision, but everything else looks perfect 👍

File: packages/quantic/force-app/main/default/lwc/quanticNumberButton/quanticNumberButton.js [hunk 3]
```diff
@@ -20,6 +20,12 @@ export default class QuanticNumberButton extends LightningElement {
    * @type {number}
    */
   @api selected;
+  /**
+   * The value for aria-label
```
Comment (mmitiche) [inline packages/quantic/force-app/main/default/lwc/quanticNumberButton/quanticNumberButton.js:24]:
```suggestion
   * The value for aria-label.
```

**Review by mmitiche — COMMENTED** [review]
Nice job!
We need to add the French translations of the new labels to this file [fr.translation-meta.xml](https://github.com/coveo/ui-kit/blob/master/packages/quantic/force-app/main/translations/fr.translation-meta.xml)

**Review by acarpentier01 — APPROVED** [review]
Looks good!👍

**Review by lbergeron — APPROVED** [review]

**Review by btaillon — APPROVED** [review]

File: packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 4]
```diff
@@ -147,6 +147,34 @@
         <protected>false</protected>
         <shortDescription>Next</shortDescription>
     </labels>
+    <labels>
+        <fullName>quantic_PreviousPage</fullName>
+        <value>Previous Page</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Previous page</shortDescription>
```
Comment (acarpentier01) [inline packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml:155]: French translation: Page précédente

File: packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 5]
```diff
@@ -147,6 +147,34 @@
...
+    </labels>
+    <labels>
+        <fullName>quantic_NextPage</fullName>
+        <value>Next Page</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Next page</shortDescription>
```
Comment (acarpentier01) [inline packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml:162]: French translation: Page suivante

File: packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 6]
```diff
@@ -147,6 +147,34 @@
...
+    </labels>
+    <labels>
+        <fullName>quantic_GoToPage</fullName>
+        <value>Go to page {{0}}</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Go to page ...</shortDescription>
```
Comment (acarpentier01) [inline packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml:169]: French translation: Aller à la page ...

File: packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 7]
```diff
@@ -147,6 +147,34 @@
...
+    </labels>
+    <labels>
+        <fullName>quantic_ShowNResultsPerPage</fullName>
+        <value>Show {{0}} number of results per page</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Show N results per page.</shortDescription>
```
Comment (acarpentier01) [inline packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml:176]: French translation: Afficher <nombre> résultats par page.

<nombre> will be replaced by a number, right? E.g.,  "Afficher 5 resultats par page."

**Review by acarpentier01 — COMMENTED** [review]
Let me know if I missed any strings. Thanks!

**Pull request merged**